website/content/commands/intention/get.mdx

```diff
@@ -7,13 +7,23 @@ page_title: 'Commands: Intention Get'
 
 Command: `consul intention get`
 
+Corresponding HTTP API Endpoint: [\[GET\] /v1/connect/intentions/exact](https://www.consul.io/api-docs/connect/intentions##read-specific-intention-by-name)
```

Comment:
There's a double `##` in the URL

Reply:
This has two corresponding HTTP API endpoints. The other is:
`[\[GET\] /v1/connect/intentions/:uuid](/api-docs/connect/intentions#read-specific-intention-by-id)`

Reply:
Fixed.